.travis.yml

```diff
@@ -87,7 +86,7 @@ before_script:
 script:
 - ./tools/lint.py
 - bash -c "sleep 2100; pkill ninja" &
-- ./tools/build.py -j2
+- ./tools/build.py --release -j2
 - ./tools/test.py $DENO_BUILD_PATH
```

Comment:
I think this will be failed becuase you used `--release` option.
I understand your intention now. But shouldn't we update `.travis.xml` and `./tools/testdata/travis_benchmark.json`? (We probably should remove $DENO_BUILD_PATH in there)

Reply:
travis_benchmark.json is just test data to make sure our scripts parse it correctly. So it doesn't matter.

Sorry, I kinda dropped the ball with this PR. I'll try to land today.